packages/core/src/signals/SignalContext.ts

```diff
@@ -257,6 +279,27 @@ export class SignalContext<
     return this.set(this.get());
   }
 
+  /**
+   * Checks if the signal is currently using its initial value.
```

Comment:
Docs for functions should use imperative present tense:
```suggestion
   * Check if the signal is currently using its initial value.
```